lib/network/udp_client.py

```diff
@@ -0,0 +1,26 @@
+import socket
+
+class UdpClient:
+    
+    def __init__(self, ip, port, buffer_size=1024):
+        self.ip = ip
+        self.port = port
+        self.buffer_size = buffer_size
+
+    def send_message(self, bytes_):
+        self.udp_client_socket.sendto(bytes_, (self.ip, self.port))
+        msg_from_server = self.udp_client_socket.recvfrom(self.buffer_size)
+        self.receive_message(msg_from_server)
+
+    def __enter__(self):
+        self.udp_client_socket = socket.socket(family = socket.AF_INET, type = socket.SOCK_DGRAM)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.udp_client_socket.close()

```

Comment:
```suggestion
    def __init__(self):
        self.udp_client_socket = socket.socket(family = socket.AF_INET, type = socket.SOCK_DGRAM)

    def dispose(self):
        self.udp_client_socket.close()
```
I looked into it and enter/exit is not what we want, so we can just do this setup on the constructor. Unfortunately, Python doesn't have a real "destructor" (`__del__` isn't guaranteed to work), so we'll call it `dispose` to match the dashboard. 

Also, it might be from my code or online, but what does `socket.AF_INET` do?